Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/inbound/PassthroughInboundDocQueryTest.java

```diff
@@ -48,13 +58,27 @@ public void passthroughInboundDocQueryHomeHcid() {
     }
 
     private void passthroughInboundDocQueryHomeHcid(String sendingHcid, String sendingHcidFormatted) {
-
         AssertionType mockAssertion = getMockAssertion(sendingHcid);
+
+        when(mockAssertion.getDeferredResponseEndpoint()).thenReturn("http://deferredEnpoint/");
```

Comment:
Why are we modifying this test? Deferred logic should only run when the endpoint is provided - logic should remain unchanged if the deferred endpoint is not a part of the assertion.

You should have a new JUnit test to test your new functionality.